modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc

```diff
@@ -235,8 +169,8 @@ kubectl edit pod <operator-name> --namespace <namespace>
 ```bash
 kubectl --namespace <namespace> patch redpanda <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
 kubectl --namespace <namespace> patch helmrelease <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmrepository <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
+kubectl --namespace <namespace> patch <namespace>-helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
```

Comment:
```suggestion
kubectl --namespace <namespace> patch helmchart <namespace>-<cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
```

## Reference

https://github.com/fluxcd/helm-controller/blob/610f6e6d38cd2c0e3b75dc6a403cc0541146c48a/api/v2beta1/helmrelease_types.go#L1149-L1152
and
https://github.com/fluxcd/helm-controller/blob/610f6e6d38cd2c0e3b75dc6a403cc0541146c48a/internal/reconcile/helmchart_template.go#L205
and
https://github.com/redpanda-data/redpanda-operator/blob/fa12e1139a687ae49e7c1e089c719c7d9c28eb1b/src/go/k8s/internal/controller/redpanda/redpanda_controller.go#L768
and
https://github.com/redpanda-data/redpanda-operator/blob/fa12e1139a687ae49e7c1e089c719c7d9c28eb1b/src/go/k8s/api/redpanda/v1alpha1/redpanda_types.go#L167-L169